roles/dhcp-server/templates/boxes.conf.j2

```diff
@@ -0,0 +1,20 @@
+# /etc/dnsmasq.d/boxes.conf
+#
+
+# defaults:
+# drive
+# tasks
+
+dhcp-option-force=209,"partman-auto/disk=/dev/{{default_install_dev}} grub-installer/bootdev=/dev/{{default_install_dev}} tasks={{default_install_tasks}}"
+
+{% for box in boxes %}
+# {{box.hostname}}
+dhcp-host={{box.mac}},set:{{box.hostname}},{{box.hostname}}{% if box.ip is defined %},{{box.ip}}{% endif %}
```

Comment:
I guess we need to namespace those tags.

Reply:
can we do it some other time when I know what that means?

Reply:
I mean, you're mixing hostnames with things like `efibc` and `bios`. What happens if I call a machine `bios`?